flink-table/pom.xml

```diff
@@ -76,10 +76,10 @@ under the License.
 	</dependencyManagement>
 
 	<properties>
-		<calcite.version>1.26.0</calcite.version>
+		<calcite.version>1.32.0</calcite.version>
 		<!-- Keep Janino in sync with calcite. -->
 		<janino.version>3.0.11</janino.version>
```

Comment:
Update Janino version to 3.1.8 (https://github.com/apache/calcite/blob/calcite-1.32.0/gradle.properties#L118)

Reply:
Before doing such update it's required to make it working...
right now it doesn't work...

Reply:
Calcite - 1.32.0 has dependency on Janino 3.1.8.  (<!-- Keep Janino in sync with calcite. -->)

Reply:
yes, i know
however from another side current there is a huge amount of tests which should be passed.
With current changes they are passing.
With janino 3.1.x no.
Once you have them passing - feel free to update the version

Reply:
Got it… Let me give a try …

Reply:
since this PR is draft I consider it mostly like a sandbox
however you might be interested in having a look at https://github.com/apache/flink/pull/21203
where janino is updating to 3.1.8